### ISSUE TITLE
Can't set orientation

### ISSUE THREAD

**Issue by steventilator** [classapp/react-native-video-helper#33]
When trying to compress a video in landscape format it gets turned into a portrait format. This changes the aspect ratio. I've tried setting the defaultOrientation to 0, 90, 180, 270, 360 and undefined. Compression works like a charm for videos in portrait mode. Am I doing anything wrong?

**Comment by ogulcantumdogan** [comment]
@steventilator  Also having the same issue, were you able to come up with a solution?

**Comment by steventilator** [comment]
@ogulcantumdogan No. I ended up using `react-native-camera` instead where I could set the `videoQuality` prop. That was a good enough solution for me.
